src/applications/verify/containers/VerifyApp.jsx

```diff
@@ -98,7 +97,7 @@ export class VerifyApp extends React.Component {
               <div>
                 <h1>Verify your identity</h1>
                 <AlertBox
-                  content={`You signed in with ${this.signInMethod}`}
+                  content={`You signed in with ${signInMethod}`}
                   isVisible
                   status="success"
```

Comment:
Since we're already changing this, can we update this to use the Web Components from Storybook.  https://design.va.gov/storybook/?path=/docs/components-va-alert--default. Should be a quick fix & you shouldn't have to import `<va-alert />`